composer.json

```diff
@@ -16,6 +16,7 @@
         "ext-reflection": "*",
         "ext-xml": "*",
         "ext-xmlreader": "*",
+        "neos/composer-plugin": "^2.0",
```

Comment:
@mhsdesign Can we revert the change in order? As this will lead to other merge conflicts on next upmerge from 7.3.

Reply:
im afraid that this is correct. I didnt do this manually but the composer manifest merger did this.
Also there will be possibly only one last upmerge from 7.3 so i guess this is all fine.

Reply:
Comes from here ( https://github.com/neos/flow-development-collection/pull/3273/files#diff-b54379d1fdcdc7a8433b179ba5a080f729e68a559cd2a6203e9d1c49b7789c3eR32 ), so (new) order should be correct, I assume someone merged a change with manually modified composer.json before, hint always run the merger for PR-ing ;)